tests/components/verisure/test_ethernet_status.py

```diff
@@ -24,7 +24,7 @@
 @contextmanager
 def mock_hub(config, response):
     """Extensively mock out a verisure hub."""
-    hub_prefix = "homeassistant.components.verisure.binary_sensor.hub"
+    hub_prefix = "homeassistant.components.verisure.verisure.VerisureHub"
```

Comment:
Not really sure how to mock this properly 😅 
@Quentame could you please help me with this? Thanks :)

Reply:
```suggestion
    hub_prefix = "homeassistant.components.verisure.VerisureHub"
```
No `ModuleNotFoundError` with this.

This is a HA class in an `__init__` file, so directly in the package.

To mock the Verisure session as just below : `verisure_prefix = "homeassistant.components.verisure.verisure.Session"`
Here you are not mocking the entire Verisure session, only the on you need for your test.

Reply:
But this is not saving the tests : 

```shell
    async def setup_verisure_locks(hass, config):
        """Set up mock verisure locks."""
        with mock_hub(config):
            await async_setup_component(hass, VERISURE_DOMAIN, config)
            await hass.async_block_till_done()
            # lock.door_lock, ethernet_status
>           assert len(hass.states.async_all()) == 2
E           assert 0 == 2
E            +  where 0 = len([])
E            +    where [] = <bound method StateMachine.async_all of <homeassistant.core.StateMachine object at 0x108863d10>>()
E            +      where <bound method StateMachine.async_all of <homeassistant.core.StateMachine object at 0x108863d10>> = <homeassistant.core.StateMachine object at 0x108863d10>.async_all
E            +        where <homeassistant.core.StateMachine object at 0x108863d10> = <homeassistant.core.HomeAssistant object at 0x108863e10>.states
```

Reply:
With:
```shell
ERROR    homeassistant.setup:setup.py:181 Error during setup of component verisure
Traceback (most recent call last):
  File "/Users/pollet/dev/home-assistant/core/homeassistant/setup.py", line 175, in _async_setup_component
    component.setup, hass, processed_config  # type: ignore
  File "/usr/local/Cellar/python/3.7.7/Frameworks/Python.framework/Versions/3.7/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/Users/pollet/dev/home-assistant/core/homeassistant/components/verisure/__init__.py", line 78, in setup
    hub.update_overview
  File "/Users/pollet/dev/home-assistant/core/homeassistant/util/__init__.py", line 208, in __call__
    and "." not in method.__qualname__.split(".<locals>.")[-1]
  File "/usr/local/Cellar/python/3.7.7/Frameworks/Python.framework/Versions/3.7/lib/python3.7/unittest/mock.py", line 600, in __getattr__
    raise AttributeError(name)
AttributeError: __qualname__
ERROR    homeassistant.core:core.py:145 Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/Users/pollet/dev/home-assistant/core/homeassistant/core.py", line 1209, in async_call
    raise ServiceNotFound(domain, service) from None
homeassistant.exceptions.ServiceNotFound: Unable to find service persistent_notification/create
```

Reply:
Looked more but didn't found yet how to fix this tests

Reply:
PS: clean your repo, there is a lot of branches 😄

Reply:
Thank you very much for looking into this.

Yeah I need to clean up my repo, thanks for the reminder 😁 I need to acquire the habit of deleting my successfully merged or closed branches.